static/js/stat_var_hierarchy/stat_var_group_node.tsx

```diff
@@ -191,31 +184,37 @@ export class StatVarGroupNode extends React.Component<
   }
 
   private fetchData(): void {
-    if (this.state.dataFetched) {
-      return;
-    }
     // stat var (group) dcid can contain [/_-.&], need to encode here.
```

Comment:
I remember there could be multiple fetchData() for a render, hence using this "if" statement to guard. Can you check if that's still valid.

Reply:
I have checked and there is only one fetchData()